pyproject.toml

```
@@ -17,3 +117,56 @@ skip_glob = []
 skip = [
     "kornia/__init__.py"
 ]
+
+[tool.pytest.ini_options]
+norecursedirs = [
+    ".git",
+    ".github",
+    "dist",
+    "build",
+]
```

I think this isn't necessary at all (`testpaths` is defined)

```suggestion
[tool.pytest.ini_options]

```

---

Let's do this in a separate PR.  Migrating and modifying at the same time make A / B comparisons problematic.